corpus/structure/CSIPSTR5/IP_18000_CSIPSTR5_1/METS.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Minimal IP with schemas -->
+<!-- In this example IDs are carring information to ease understanding - they look like paths to ease understanding, but are just IDs, similar to the naming of namespaces - these IDs can be replaced with information less UUID -->
+<!-- CSIPSTR15 goes like this: We recommend including all schema documents for any structured metadata within package. These schema documents SHOULD be placed in a sub-folder called schemas within the Information Package root folder and/or the representation folder. -->
+<mets xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+  xmlns="http://www.loc.gov/METS/" 
+  xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance schemas/XMLSchema.xsd http://www.loc.gov/METS/ schemas/mets.xsd http://www.w3.org/1999/xlink schemas/xlink.xsd https://DILCIS.eu/XML/METS/CSIPExtensionMETS schemas/CSIPExtensionMETS.xsd" 
+  OBJID="Minimal_IP_with_schemas" 
```

Comment:
wrong objid

Reply:
Related to `mets/@OBJID`, according to CSIP86, `mets/structMap[@LABEL='CSIP']/div/@LABEL` must also match the name of the package root folder.